include/solvers/navier_stokes_base.h

```diff
@@ -899,11 +899,11 @@ class NavierStokesBase : public PhysicsSolver<VectorType>
   /**
    * @brief Gather solution information to generate output results
    *
-   * @param[in] solution Vector of present solution
+   * @param[in] solution Vector of the present solution
    * @param[in,out] solution_output_structs Vector of OutputStructs that will be
    * used to write the output results as VTU files
    */
-  void
+  virtual void
```

Comment:
There is no need to make the function virtual. It already has a gather output results hook that serves this purpose instead of rewriting the entire function
```suggestion
  void
```